code/modules/projectiles/guns/hookshot.dm

```diff
@@ -506,3 +506,140 @@
 	icon_state = "vampkiller_true"
 	hooktype = /obj/item/projectile/hookshot/whip/vampkiller/true
 
+//Windup-Boxes/////////////////////////////////////////////////////
+
+/obj/item/weapon/gun/hookshot/whip/bootbox
+	name = "boot-in-a-box"
+	icon = 'icons/obj/wind_up.dmi'
+	icon_state = "bootbox-0"
+	item_state = "bootbox"
+	desc = "A box with a spring-loaded boot inside. There is a crank attached to wind it up."
+	fire_action = "activate"
+	inhand_states = list("left_hand" = 'icons/mob/in-hand/left/guns_experimental.dmi', "right_hand" = 'icons/mob/in-hand/right/guns_experimental.dmi')
+	clumsy_check = 0 //Just makes sense
+	force = 5
+	maxlength = 4 //whips are weird and this ends up being 2
```

Comment:
instead, make a define like this
`#define WHIPLENGTH * 2`

then do this as `maxlength = 2 WHIPLENGTH` - that makes it immediately evident what this is without need for a comment

Reply:
That was more for people judging balance/myself to remember. It has 4 range like the bullwhip has 5 but it counts the tile it was fired on and one right after the projectile ends. So the bullwhip travels 3. That doesn't actually do anything.

Reply:
then define whiplength as `+2`